.gitignore

```diff
@@ -3,3 +3,4 @@ phpunit.xml
 composer.lock
 composer.phar
 vendor/*
+.idea/
```

Comment:
You can remove the line locally, add it, amend your commit and do a forced push like this:

```
git commit --amend
git push -f
```